README.md

```diff
@@ -34,7 +34,10 @@ the native networking implementation – but with higher security,
 
 To get started, you first need a working installation of
 [OpenStack](http://www.openstack.org/). Then download and install the latest
-stable build of Calico [here](http://www.projectcalico.org/download/).
+stable build of Calico following the instructions [here](http://www.projectcalico.org/download/).
+
+Technical documentation is [here](https://github.com/Metaswitch/calico-docs/wiki); if you are going to contribute to the project, you'll also need to run the
+[tests](doc/CalicoUTs.md).
```

Comment:
I agree the punctuation and spacing is a bit odd, but I didn't write or change most of it! I've cleaned up though.